src/main/java/com/cloudbees/jenkins/support/impl/JenkinsLogs.java

```diff
@@ -458,6 +465,15 @@ public static boolean isRequired() {
             }
         }
 
+        public static Future<List<LogRecord>> getLogRecords(@NonNull VirtualChannel channel) throws IOException {
+            return channel.callAsync(new SlaveLogFetcher());
```

Comment:
Some data is: https://github.com/jenkinsci/remoting/blob/master/src/main/java/hudson/remoting/Channel.java#L157.